docusaurus.config.js

```diff
@@ -272,24 +272,17 @@ const config = {
           {
             to: "docs",
             position: "left",
-            label: "Getting started",
```

Comment:
Can we switch "Get Started" and "Learn" so that learn is first? Now that it's so sexy!

Reply:
Yeah something I want to explore is to have 

Install | Get started | (other links). 

Similar to https://www.rust-lang.org/learn

I want to make a custom page for the install steps, simplifying it and making it look sexy too, 

Thoughts?

Reply:
@eladb I made the new install page WYT? https://wing-docs-git-new-learn-section-for-website-monada.vercel.app/install

Hopefully step by step this will tidy up the process.

Next thing is the Wing book I think

Reply:
I think we can merge this now as I focus on the the detailed content. Will merge!